platform/android/detect.py

```diff
@@ -80,7 +85,7 @@ def install_ndk_if_needed(env: "SConsEnvironment"):
     sdk_root = env["ANDROID_HOME"]
     if not os.path.exists(get_android_ndk_root(env)):
         extension = ".bat" if os.name == "nt" else ""
-        sdkmanager = sdk_root + "/cmdline-tools/latest/bin/sdkmanager" + extension
+        sdkmanager = escape_space_in_path(sdk_root + "/cmdline-tools/latest/bin/sdkmanager" + extension)
```

Comment:
I think it would be more readable to use f-Strings directly, improving the concatenation too.

Reply:
For all the paths or just here?

Reply:
To be consistent, use it for all paths.

Reply:
Have made the changes. Kindly let me know if there are any further changes required.